spec/controllers/assets_controller_spec.rb

```diff
@@ -206,6 +213,24 @@
         expect(assigns(:asset).access_limited).to eq([])
       end
 
+      it 'stores auth_bypass_ids on existing asset' do
+        attributes = valid_attributes.merge(auth_bypass_ids: ['bypass-id'])
+        put :update, params: { id: asset.id, asset: attributes }
+
+        expect(assigns(:asset).auth_bypass_ids).to eq(['bypass-id'])
+      end
+
+      it 'copes when auth_bypass_ids are passed in as an empty string' do
+        asset.update_attributes!(auth_bypass_ids: %w[bypass-1 bypass-2])
```

Comment:
Is this line needed?

Reply:
Yup, as by default an asset will have an empty array as auth_bypass_ids and thus this is set to a populated array to test that it is modified to be an empty array